owncloudData/src/main/java/com/owncloud/android/data/storage/LocalStorageProvider.kt

```diff
@@ -58,9 +58,16 @@ sealed class LocalStorageProvider(private val rootFolderName: String) {
      * file.
      */
     fun getDefaultSavePathFor(
-        accountName: String?,
-        remotePath: String
-    ): String = getAccountDirectoryPath(accountName) + remotePath
+        accountName: String,
+        remotePath: String,
+        spaceId: String?,
```

Comment:
Maybe setting this parameter with default value `null`? Just like in `getTemporalPath`

Reply:
No default value is needed for this one. We want to make sure that we receive the spaceId, even if it's null.

I would change the one from `getTemporalPath` though